wilma/server_mock/server.js

```diff
@@ -0,0 +1,186 @@
+'use strict';
+
+let express = require('express');
+let favicon = require('serve-favicon');
```

Comment:
Wird nicht verwendet

```suggestion
let express = require('express');
```